signature-aggregator/aggregator/cache/cache.go

```diff
@@ -0,0 +1,54 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"go.uber.org/zap"
+)
+
+const (
+	initialCapacity = 1024
+	maxSize         = 1024 * 1024
```

Comment:
MaxSize should be configurable via the application configs. We should have a sane default though

Reply:
done. I'm not sure what would be sane to you though. I left this number in place. Happy to change it to whatever the team thinks.